services/m3dbnode/server/server.go

```diff
@@ -790,3 +835,60 @@ func capacityPoolOptions(
 	}
 	return opts
 }
+
+func createStaticShardSet(numShards int, listenAddress string) (sharding.ShardSet, []topology.HostShardSet, error) {
```

Comment:
nit: To keep consistent with our constructor-like methods being named with "new" probably should rename this to `newStaticShardSet(...)`.